pyopendoc/writer.py

```diff
@@ -75,6 +75,27 @@ def write_to_table(self, table_name, column, row, value, refresh=False, target=N
 
         span.text = value
 
+    def write_to_frame(self, frame_name, x_value, y_value, value=None, target=None, unit="in"):
+        """Writes to x, y coordinate of a specified frame.
+        Defaults unit to inches.
+
+        """
+        target_file = target if target else self.CONTENT_FILE
+        xml_file = self.get_file(target_file)
+        if xml_file.filetype != "XML":
+            raise TypeError("Not an XML file!")
+
+        for df in xml_file.root.findall(".//draw:frame", self.NAMESPACES):
+            frame = df.get("{%s}name" % self.NAMESPACES["draw"])
+            if frame == frame_name:
```

Comment:
You could have extracted the logic for finding the frame out into its own separate function called something like `find_frame`, and returned the frame from that. Doing so is generally considered good practice, as it reduces the effort needed to read and understand your code (often described as "cognitive load").

It isn't bad form to return nothing if you logically have nothing to return. You can use return to terminate a function early without actually returning anything, simply by supplying the `return` keyword with no arguments. But if you want, you could return a value signifying whether the operation was successful or not.